enclave-manager/web/src/components/DownloadButton.tsx

```diff
@@ -1,10 +1,11 @@
 import { Button, ButtonProps, IconButton, IconButtonProps } from "@chakra-ui/react";
 import { FiDownload } from "react-icons/fi";
-import { isDefined } from "../utils";
+import streamsaver from "streamsaver";
```

Comment:
@Dartoxian do you need to add this guy to `package.json`?

Reply:
no it's already there.